website/src/index.css

```diff
@@ -7,7 +7,7 @@
   --ion-color-primary-shade: #17549a;
   --ion-color-primary-tint: #3170b7;
 
-  --ion-color-secondary: #001531;
+  --ion-color-secondary: #131415;
```

Comment:
Why did you change this color? This change reflects on other pages, this must be discussed before applied!

Reply:
I put it back as before